static/templates/settings/notification_settings.hbs

```diff
@@ -75,9 +75,9 @@
                     <option value="{{ this }}">{{ this }}</option>
                     {{/each}}
                 </select>
-                <a id="play_notification_sound">
-                    {{t "Play sound" }}
-                </a>
+                <span id="play_notification_sound">
+                    <i class="fa fa-play-circle" aria-hidden="true"></i>
```

Comment:
aria-hidden seems wrong here; this should have an `aria-label` instead.  (I.e. this widget should be a thing blind users using screenreaders can interact with).  aria-hidden should just be on decorative icons next to the text label for the item (E.g. in many of our menus).